sceptre/plan/plan.py

```diff
@@ -440,3 +440,13 @@ def dump_config(self, *args):
         """
         self.resolve(command=self.dump_config.__name__)
         return self._execute(self.config_reader, *args)
+
+    def dump_template(self, *args):
+        """
+        Returns a generated Template for a given Stack
+
+        :returns: A dictionary of Stacks and their template body.
+        :rtype: dict
+        """
+        self.resolve(command=self.generate.__name__)
```

Comment:
Why is this calling `generate`? I think that might be a mistake.

Reply:
fixed.